udata/core/site/views.py

```diff
@@ -163,7 +163,13 @@ def get_context(self):
         return context
 
 
+@blueprint.route('/terms/')
```

Comment:
It could but I don't think it's really needed and/or worth the complexity for now.
If you don't need it and someone needs it, it can be done in another PR.